### PR TITLE
Rgw fixes

### PR DESCRIPTION
**Pull request by leseb**

### CODE REVIEW

File: roles/ceph-rgw/handlers/main.yml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
 - name: update period
-  command: radosgw-admin period update --commit
+  command: radosgw-admin --cluster {{ cluster }} period update --commit
```
Comment (guits) [inline roles/ceph-rgw/handlers/main.yml]: this command will be run on first mon node as many rgw nodes there is on the cluster, is that expected ?

File: roles/ceph-rgw/handlers/main.yml [hunk 2]
```diff
@@ -1,6 +1,7 @@
...
+  delegate_to: "{{ groups[mon_group_name][0] }}"
```
Comment (guits) [inline roles/ceph-rgw/handlers/main.yml]: `run_once: true` ?

Comment (leseb) [reply roles/ceph-rgw/handlers/main.yml]: affirmatif

Comment (guits) [reply roles/ceph-rgw/handlers/main.yml]: ok maggle

**Pull request merged**